AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -493,6 +494,11 @@ define([
                     });
                 }.bind(this)).fail(function (error) {
                 deferred.reject(error);
+                $(errorMessageSelector).get(0).scrollIntoView({
```

Comment:
Depending on how `licenseProcess` method will be used the error may or may not be shown to the user, also the selector of error message depends on the view layer implementation. Please move this action to the place where the error is actually added to the content.